fuel-asm/src/args.rs

```diff
@@ -248,6 +262,51 @@ crate::enum_try_from! {
 
         /// Set `$rA` to `tx.policies[count_ones(0b11111 & tx.policyTypes) - 1].expiration`
         PolicyExpiration = 0x505,
+
+        /// Set `$rA` to `Memory address of tx.root`
+        UploadRoot = 0x600,
+
+        /// Set `$rA` to `tx.witnessIndex`
+        UploadWitnessIndex = 0x601,
+
+        /// Set `$rA` to `tx.subsectionIndex`
+        UploadSubsectionIndex = 0x602,
+
+        /// Set `$rA` to `tx.subsectionsNumber`
+        UploadSubsectionsCount = 0x603,
+
+        /// Set `$rA` to `tx.proofSetCount`
+        UploadProofSetCount = 0x604,
+
+        /// Set `$rA` to `Memory address of tx.proofSet[$rB]`
+        UploadProofSetAtIndex = 0x605,
+
+        /// Set `$rA` to `Memory address of tx.id`
+        BlobId = 0x700,
+
+        /// Set `$rA` to `tx.witnessIndex`
+        BlobWitnessIndex = 0x701,
```

Comment:
I think we need better comments here=)

Reply:
I don't really see what information should be explained here. I tried to follow the current information it's not a memory address or anything. Maybe adding that is blob specific ? But the name seems straight forward. Maybe explaning what is the witness index but in this case we should explain what is each field fetch by the other GTF